packages/esm-patient-medications-app/src/utils/launchOrderBasket.tsx

```diff
@@ -0,0 +1,21 @@
+import {
+  launchPatientWorkspace,
+  launchStartVisitPrompt,
+  useVisitOrOfflineVisit,
+} from '@openmrs/esm-patient-common-lib';
+import { useCallback, useMemo } from 'react';
+
+export function useLaunchOrderBasket(patientUuid) {
+  const { currentVisit } = useVisitOrOfflineVisit(patientUuid);
+
+  const launchOrderBasket = useCallback(() => {
```

Comment:
Instead of using two hooks (`useMemo` and `useCallback`) I'd recommend to only use one (`useMemo`) depending on `currentVisit` - which already returns the callback wrapped. This way, you save a bit of perf., but also make the intention of returning the object instead of the function more clear.

Reply:
Thanks a lot @FlorianRappl !